torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -580,6 +580,30 @@ RegisterOperators reg({
         };                                                                     \
       }),
 
+#define DEFINE_INT_FLOAT_OP(aten_op, op, result)                               \
+  Operator(                                                                    \
+      #aten_op "(int a, float b) -> " #result,                                 \
+      [](const Node* node) {                                                   \
+        return [=](Stack& stack) {                                             \
+          int64_t a;                                                           \
+          double b;                                                            \
+          pop(stack, a, b);                                                    \
+          push(stack, op);                                                     \
+          return 0;                                                            \
+        };                                                                     \
+      }),                                                                      \
+  Operator(                                                                    \
+      #aten_op "(float a, int b) -> " #result, [](const Node* node) {          \
+        return [=](Stack& stack) {                                             \
+          double a;                                                            \
+          int64_t b;                                                           \
+          pop(stack, a, b);                                                    \
+          push(stack, op);                                                     \
+          return 0;                                                            \
+        };                                                                     \
+      }),
+
+
 #define DEFINE_INT_OP(aten_op, op)                            \
```

Comment:
Actually, can you merge the DEFINE_INT_FLOAT_OP and DEFINE_INT_OP? this will make use having less macros that is hard to read about